immersion/kubeflow_pipelines/cicd/solutions/cloudbuild_vertex.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+
+# https://www.apache.org/licenses/LICENSE-2.0
+    
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either 
+# express or implied. See the License for the specific language governing 
+# permissions and limitations under the License.
+
+steps:
+# Build the trainer image
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/trainer_image_covertype_vertex:latest', '.']
+  dir: trainer_image_vertex
+  
+
+# Compile the pipeline
+- name: 'gcr.io/$PROJECT_ID/kfp-cli-vertex'
+  args:
+  - '-c'
+  - |
+    dsl-compile-v2 --py pipeline.py --output covertype_kfp_pipeline.json
+  env:
+  - 'PIPELINE_ROOT=gs://$PROJECT_ID-vertex/pipeline'
+  - 'PROJECT_ID=$PROJECT_ID'
+  - 'REGION=$_REGION'
+  - 'SERVING_CONTAINER_IMAGE_URI=us-docker.pkg.dev/vertex-ai/prediction/sklearn-cpu.0-20:latest'
+  - 'TRAINING_CONTAINER_IMAGE_URI=gcr.io/$PROJECT_ID/trainer_image_covertype_vertex:latest'
+  - 'TRAINING_FILE_PATH=gs://$PROJECT_ID-vertex/data/training/dataset.csv'
+  - 'VALIDATION_FILE_PATH=gs://$PROJECT_ID-vertex/data/validation/dataset.csv'
+  dir: pipeline_vertex
+  
+# Run the pipeline
+- name: 'gcr.io/$PROJECT_ID/kfp-cli-vertex'
+  args:
+  - '-c'
+  - |
+    python kfp-cli_vertex/run_pipeline.py --project_id=$PROJECT_ID --template_path=covertype_kfp_pipeline.json --display_name=coverype_kfp_pipeline --region=$_REGION
```

Comment:
this i think needs to be `--template_path=pipeline_vertex/covertype_kfp_pipeline.json` to match the `dir` above, otherwise the build fails.